packages/Notification/Notification.jsx

```diff
@@ -32,7 +32,7 @@ const renderContent = (variant, children) => {
 
   if (variant === 'error') {
     return (
-      <ColoredTextProvider colorClassName={messagingStyles.errorText}>
+      <ColoredTextProvider colorClassName={messagingStyles.errorTextNotification}>
```

Comment:
The `ColoredTextProvider` was invented as a sneaky way of changing text color without changing the public API of the Typography components.

If the Notification is now just going to use the default text color, then there is no more need for the ColoredTextProvider component in this case, and you maybe can just remove its use from Notification altogether.

`color-shark` is just the normal `color-text`. https://github.com/telusdigital/tds/blob/master/packages/colours/_colours.scss#L21